app/models/policies/early_career_payments/eligibility.rb

```diff
@@ -79,6 +79,7 @@ def self.max_award_amount_in_pounds
 
       validates :nqt_in_academic_year_after_itt, on: [:"nqt-in-academic-year-after-itt", :submit], inclusion: {in: [true, false], message: "Select yes if you are currently teaching as a qualified teacher"}
       validates :current_school, on: [:"correct-school"], presence: {message: "Select the school you teach at or choose somewhere else"}, unless: :school_somewhere_else?
+      # TODO: Does this need to be removed now it has been moved to a Form object?
```

Comment:
Steve asked me to remove the validation from my PR so I think it's best to remove it